packages/venia-concept/src/components/PurchaseHistoryPage/PurchaseHistory/purchaseHistoryContainer.js

```diff
@@ -3,14 +3,10 @@ import { transformItems } from 'src/selectors/purchaseHistory';
 import PurchaseHistory from './purchaseHistory';
 import actions, { getPurchaseHistory } from 'src/actions/purchaseHistory';
 
-const mapStateToProps = ({ purchaseHistory }) => {
-    let { isFetching, items } = purchaseHistory;
-
-    items = transformItems(items);
-
+const mapStateToProps = ({ purchaseHistory: { isFetching, items } }) => {
```

Comment:
Please double check my changes here!

Reply:
Nested destructuring is too difficult to read for too little benefit. I recommend reverting this change.

Reply:
Resolved in https://github.com/magento-research/pwa-studio/pull/1088/commits/c12f9179443d38f9c434dfbfda04872e2033dad6.